src/eventcontroller/EventController.ts

```diff
@@ -23,4 +41,5 @@ export default interface EventController {
    * Adds a state to the meeting history stack.
```

Comment:
Can we improve this docstring? i.e. I don't know what this does.

Reply:
Updated